### PR TITLE
More clarification

### PR DESCRIPTION
**Pull request by joinimran**
There was confusion if the license is not assigned to the user. Updated the information.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9161

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/defender-endpoint/mac-support-license.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Contact your administrator for help.
 
 **Cause:** 
 
-You deployed and/or installed the Microsoft Defender for Endpoint on macOS package ("Download installation package") but you might have run the configuration script ("Download onboarding package").
+You deployed and/or installed the Microsoft Defender for Endpoint for macOS package ("Download installation package") but you might have run the configuration script ("Download onboarding package"). Or You have not assigned the license to the user.
```
Comment (ghost) [inline microsoft-365/security/defender-endpoint/mac-support-license.md]: Very well. Let us go for the same line as in your previous PR - https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9251:
```suggestion
You deployed and/or installed the Microsoft Defender for Endpoint for macOS package ("Download installation package"), but you might have run the configuration script ("Download onboarding package"), or you have not assigned a license to the user.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - only @illfated's suggestions.

**Comment by nam31** [comment]
@Dansimp PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by ghost** [comment]
@joinimran : Thank you for recreating your PR in the new repository. This PR will be ready to merge as soon as the suggested change has been committed.

**Pull request merged**